cmd/seed-e2e/main.go

```
@@ -0,0 +1,37 @@
+
+import (
+	"factorbacktest/internal/util"
+	"factorbacktest/tools/seeds"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+)
+
+func main() {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
```

golangci-lint (2.11.4)

[error] Error return value of `logger.Sync` is not checked

(errcheck)